package.json

```diff
@@ -11,12 +11,13 @@
     "debug": "node --max_old_space_size=4096 --inspect-brk ./dist/src/index.js",
     "eslint-fix": "eslint . --ext .ts --ignore-pattern node_modules/ --ignore-pattern dist/ --fix",
     "eslint": "eslint . --ext .ts --ignore-pattern node_modules/ --ignore-pattern dist/",
-    "build": "tsc -p .",
+    "build": "tsc && cp -R src/templates dist/src",
```

Comment:
Could you add eslint into the build?

Reply:
This already an indenpendent task for eslint.

Reply:
we need to make sure the PR with eslint issues should not be merged.

Reply:
The PR check is against the whole CI tasks. No PR will be merged if one of these task fails.

Reply:
changed now.